### PR TITLE
feat: add emerging-threat rules related to mint-sandstorm

### PR DESCRIPTION
**Pull request by nasbench**
### Summary of the Pull Request

This PR adds new rules based on MSTIC reporting https://www.microsoft.com/en-us/security/blog/2023/04/18/nation-state-threat-actor-mint-sandstorm-refines-tradecraft-to-attack-high-value-targets/

### Detailed Description of the Pull Request / Additional Comments

N/A

### Example Log Event

N/A

### Fixed Issues

NA

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml
Comment (phantinuss) [inline rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml:112]: the test_rules.py doesn't work for this folder. Also there is no `selection_parent_path` in this rule. :)

Comment (phantinuss) [reply rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml:112]: We have to rework the repo a bit. I just started thinking about the solution. Seems to me it the straight forward solution is to duplicate the workflow action and run a separate job for the emerging threats folder

Comment (nasbench) [reply rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml:112]: I actually reworked the tests here https://github.com/SigmaHQ/sigma/pull/4161
I won't merge this if flo doesn't merge it now. I'm still migrating the older rules.

Comment (nasbench) [reply rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml:112]: Don't mind the shit show that happened when I did a git pull from master (big mistake)

Comment (phantinuss) [reply rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml:112]: did you do a --rebase pull? imo that's the preferred way to do so.

You cannot merge yet, you have the error in the condition of the first rule. one moment

Comment (phantinuss) [inline rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml:112]:
```suggestion
    condition: selection_parent and (all of selection_special_child_powershell_* or all of selection_special_child_lsass_* or 1 of selection_child_*)
```
this was a copy paste thingy, I guess

Comment (nasbench) [reply rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml]: Sadly yes.

**Review by phantinuss — APPROVED** [review]

File: rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml
Comment (nasbench) [reply rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml]: Alright a rebase has done the job. Cheers.

Comment (nasbench) [reply rules-emerging-threats/2023/Mint-Sandstorm/proc_creation_win_apt_mint_sandstorm_aspera_faspex_susp_child_process.yml]: Other PR merged. We'll merge this.

**Pull request merged**
